### ISSUE TITLE
Minor annoyance: Onivim Installation and first execution after installation blocked by AV

### ISSUE THREAD

**Issue by iilonmasc** [onivim/oni2#3040]
My AV software (Avast) blocks the installer for a short period of time because it seems untrusted.
After a few minutes of checking the AV unblocks the installer, so it's not that bad.

However always needing to wait 15-20 minutes just to install onivim is a bit annoying.

The same happens when I first start onivim after an installation.
As above, after 15-20 minutes waiting for the scan to finish, consecutive startups don't face this issue, but this is annoying as well.

Is there a way to sign the binaries or something to make AVs trust onivim?

**Comment by bryphe** [comment]
Thanks for reporting the issue, @iilonmasc ! I was able to reproduce this downloading the AVG installer.

Our release binaries on the portal are actually code-signed, but it seems Avast/AVG has their own process for validating executables. Seems like we're hitting the same issue other publishers hit here: https://forum.avast.com/index.php?topic=202088.0

I've submitted the installer / zip to their false-positive-form: https://www.avast.com/false-positive-file-form.php - hopefully this addresses the issue 🤞 It is indeed annoying to have to wait every time!
